lib/Video/Adapter/Ffmpeg.php

```diff
@@ -253,7 +253,7 @@ protected function getVideoInfo()
     }
 
     /**
-     * @return float
+     * @return float|null
```

Comment:
Should also be changed in the abstract method in the parent class

Reply:
also done